clients/python/client/osparc/info/info.py

```diff
@@ -0,0 +1,12 @@
+import json
+from pathlib import Path
+from typing import Any, Dict
+
+
+def get_api() -> Dict[str, Any]:
```

Comment:
I would name this `def openapi()` like fastapi does for the same output

Reply:
done